tests/test_wsgi.py

```diff
@@ -75,10 +74,9 @@ def test_environ_headers(self):
         self.assertEqual(environ['HTTP_X_TEST'], '123,456')
         self.assertEqual(environ['SCRIPT_NAME'], 'script')
         self.assertEqual(environ['SERVER_NAME'], 'python.org')
-        self.assertEqual(environ['SERVER_PORT'], '443')
+        self.assertEqual(environ['SERVER_PORT'], '80')
```

Comment:
Port has changed because old version was trying to determine port number by url scheme. (see line 71 is_ssl=True)

So in old version it was '80' if url_scheme == 'http' else '443' (see old version aiohttp/wsgi.py:95-98)

But there is possible situations:

 http://example.com:8080/
 https://example.com:4443/

Old version will give wrong result in both cases.

Ill add more tests to cover it.